sceptre/hooks/cmd.py

```diff
@@ -19,8 +19,16 @@ def run(self):
         :raises: subprocess.CalledProcessError
         """
         envs = self.stack.connection_manager.create_session_environment_variables()
+
+        if isinstance(self.argument, str):
+            args = self.argument
+            executable = None
+        elif isinstance(self.argument, dict):
+            args = self.argument["args"]
+            executable = self.argument["executable"]
+
         try:
-            subprocess.check_call(self.argument, shell=True, env=envs)
+            subprocess.check_call(args, shell=True, env=envs, executable=executable)
         except TypeError:
             raise InvalidHookArgumentTypeError(
```

Comment:
The error message needs to change to reflect the new options. And what type of inputs will be invalid now? I hope after I learn about the unit tests it will be clear how to rewrite this.